tests/conftest.py

```diff
@@ -8,6 +8,15 @@
 from kili.core.graphql.graphql_client import GraphQLClient
 
 
+class MockResponse:
```

Comment:
MockRequestsReponse

and add type hints

Reply:
this is not a Mock but a Fake